packages/core/useSpeechRecognition/index.ts

```diff
@@ -94,6 +96,7 @@ export function useSpeechRecognition(options: SpeechRecognitionOptions = {}) {
 
     recognition.onend = () => {
       isListening.value = false
+      if (isRef(lang) && recognition) recognition.lang = lang.value
```

Comment:
```js
if (recognition)
  recognition.lang = unref(lang)
```

Reply:
For this - we only want to update `recognition.lang` if a ref is passed in - no point doing this if a plain string is used?

Reply:
👍